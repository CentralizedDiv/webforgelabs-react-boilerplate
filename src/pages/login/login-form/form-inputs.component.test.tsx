import React from 'react';
import FormInputs from './form-inputs.component';
import { shallow } from 'enzyme';

describe('Login Form Inputs', () => {
  const antdForm = { getFieldDecorator: jest.fn() };
  const props: any = {
    form: antdForm,
    password: '',
    remember: false,
    username: '',
  };
  const component = shallow(<FormInputs {...props} />);

  it('renders properly', () => {
    expect(component).toMatchSnapshot();
  });

});