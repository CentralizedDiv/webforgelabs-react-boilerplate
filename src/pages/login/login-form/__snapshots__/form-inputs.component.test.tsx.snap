// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login Form Inputs renders properly 1`] = `
<FormInputs
  form={
    Object {
      "getFieldDecorator": [MockFunction],
    }
  }
  password=""
  remember={false}
  username=""
/>
`;
